packages/iTwinUI-react/src/core/Tooltip/Tooltip.test.tsx

```diff
@@ -22,7 +22,7 @@ it('should toggle the visibility of tooltip on hover', () => {
 
   fireEvent.mouseLeave(getByText('Hover Here'));
   jest.runAllTimers();
-  expect(queryByText('some text')).not.toBeVisible();
+  expect(queryByText('some text')).not.toBeInTheDocument();
```

Comment:
this is interesting...

Reply:
![image](https://user-images.githubusercontent.com/81580355/162416151-da5be534-80ad-4a2b-b538-ad54bd599b90.png)

Why we didnt get it before? We need to carefully review our tests using `toBeVisible` cause we might get false results now :/
Or tippy behavior now somehow changed.. dont really understand

Reply:
I think tippy first becomes invisible then leaves the document. I'm not sure what the best way to test would be. My goal was to just fix all those failures in 10 different places, and I didn't find one single way to work everywhere.